system/host_cpu.tf

```diff
@@ -0,0 +1,47 @@
+resource "datadog_monitor" "cpu_average" {
```

Comment:
what do you think of making the varkiables for all the monitors as `map`?

when we can just do something like `cpu_average.cpu_time` and `cpu_age.critical_threshold`